### PR TITLE
Fix mastodon link on about page

### PR DESCRIPTION
**Pull request by schmidt**
Add TLS and remove trailing quote

### CODE REVIEW

File: content/about.md [hunk 1]
```diff
@@ -14,12 +14,12 @@ I am also trying to understand our beautiful world a little better and [this is
...
 
 #### Feel free to contact me:
 
 mail: [hello@michaelbumann.com](mailto:hello@michaelbumann.com)  
 phone: +49176 22747893 (signal/whatsapp)  
 in person: [where.is.michaelbumann.com](http://where.is.michaelbumann.com)
 
-PGP fingerprint: 18F1 CDD3 E5DE E065 20B4 C203 7B03 B936 9716 70EC"
\ No newline at end of file
+PGP fingerprint: 18F1 CDD3 E5DE E065 20B4 C203 7B03 B936 9716 70EC"
```
Comment (schmidt) [inline content/about.md:25]: Sorry. This change was unintended and caused by GitHub's web ui.

**Comment by bumi** [comment]
oh wow, how cool!! thanks a lot!! :)

**Comment by schm** [comment]
Yay 🎉  one more contribution for #hacktoberfest 😄

**Comment by schmidt** [comment]
Yay 🎉  one more contribution for #hacktoberfest 😄

**Comment by bumi** [comment]
:D :rocket:

**Pull request merged**